enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/context/HeartbeatContextImpl.java

```diff
@@ -60,11 +57,12 @@
     {
         super( me, commonState, logging, timeouts );
         this.executor = executor;
+        this.heartBeatListeners = new Listeners<>();
```

Comment:
Why do we initialise in constructor and not at declaration (as we do in the rest of this commit)?